source/trackers.json

```diff
@@ -272,6 +272,7 @@
         "graph.facebook.com": "facebook_audience",
         "solaredge.com": "solaredge",
         "crashlytics.com": "crashlytics",
+        "cloudflare-dns.com": "cloudflare",
```

Comment:
Shouldn't there be more info for `cloudflare` like `crashlytics`? I mean "name", "category" etc.

Reply:
This information is exist in the `whotracksme` data

Reply:
Ok, I've got it.